packages/frontend/src/features/sqlRunner/components/SaveSqlChartModal.tsx

```diff
@@ -153,61 +230,41 @@ export const SaveSqlChartModal: FC<Props> = ({ opened, onClose }) => {
             title={
                 <Group spacing="xs">
                     <MantineIcon icon={IconChartBar} size="lg" color="gray.7" />
-                    <Text fw={500}>Save chart</Text>
+                    <Text fw={500}>Save Chart</Text>
                 </Group>
             }
             styles={(theme) => ({
                 header: { borderBottom: `1px solid ${theme.colors.gray[4]}` },
                 body: { padding: 0 },
             })}
         >
-            <form onSubmit={form.onSubmit(handleOnSubmit)}>
-                <Stack p="md">
-                    <Stack spacing="xs">
-                        <TextInput
-                            label="Chart name"
-                            placeholder="eg. How many weekly active users do we have?"
-                            required
-                            {...form.getInputProps('name')}
-                        />
-                        <Textarea
-                            label="Description"
-                            {...form.getInputProps('description')}
-                        />
-                    </Stack>
-                    <SaveToSpace
-                        form={form}
-                        spaces={spaces}
-                        projectUuid={projectUuid}
-                    />
-                </Stack>
-
-                <Group
-                    position="right"
-                    w="100%"
-                    sx={(theme) => ({
-                        borderTop: `1px solid ${theme.colors.gray[4]}`,
-                        bottom: 0,
-                        padding: theme.spacing.md,
-                    })}
-                >
-                    <Button
-                        onClick={onClose}
-                        variant="outline"
-                        disabled={isCreatingSavedSqlChart}
+            {showWarning ? (
+                <Box>
+                    <SqlQueryBeforeSaveAlert />
```

Comment:
if there are unrun changes, alert the user, but let them save the latest `sql` regardless.